src/main/scala/mesosphere/marathon/util/heartbeat/HeartbeatActor.scala

```diff
@@ -0,0 +1,97 @@
+package mesosphere.marathon.util.heartbeat
```

Comment:
The heartbeat actor really doesn't have any scheduler integration code, it's completely generic - so that's why it lives where it currently does. `MesosHeartbeatMonitor` is a `Scheduler` implementation and so .. since we don't have a proper `core` package for scheduler stuff yet, I just put it next to the other scheduler stuff.

You're the 4th person to suggest that I couple these classes more closely. Since it seems intuitive to the majority ... I'll create `core/heartbeat` to house these.